src/chapters/ldap-auth.xml

```diff
@@ -384,6 +384,16 @@ dn: cn={4}guacConfigGroup,cn=schema,cn=config
                                 have no effect on Guacamole configurations.</emphasis></para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term><property>ldap-dereference-aliases</property></term>
+                    <listitem>
+                        <para>Controls whether or not the LDAP connection follows (dereferences) aliases
+                            as it searches the tree.  Possible values for this property are "never" (the default)
+                            so that aliases will never be followed, "searching" to dereference during search operations
+                            after the base object is located, "finding" to dereference in order to locate the
+                            search base, but not during the actual search, and "always" to always dereference aliases.
```

Comment:
Your `<para>` is missing a corresponding end tag, which is causing the manual build to fail with these changes present:

```
$ make
cd src; xsltproc -o ../html/ --xinclude site.xslt gug.xml
chapters/ldap-auth.xml:395: parser error : Opening and ending tag mismatch: para line 390 and listitem
                    </listitem>
                               ^
```